.gitignore

```diff
@@ -0,0 +1,123 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
@jdaaph Could you choose a slightly more conservative `.gitignore`? Maybe compare what freud and signac do, for reference. At least cut out all the stuff that is irrelevant (e.g. Flask, Django, celery, Sage, Rope, Scrapy, ...). Keep things like packaging and `.ipynb_checkpoints`.